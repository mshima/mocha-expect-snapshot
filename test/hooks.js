const expect = require('expect');

describe('hooks', function () {
  describe('toMachSnapshot()', function () {
    it('should match snapshot', function () {
      expect({ foo: 'bar' }).toMatchSnapshot();
    });
  });
});
