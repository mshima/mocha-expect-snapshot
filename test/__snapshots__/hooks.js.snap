// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`hooks toMachSnapshot() should match snapshot 1`] = `
Object {
  "foo": "bar",
}
`;
